contrib/runners/mistral_v2/query/mistral_v2.py

```diff
@@ -101,11 +101,13 @@ def _get_workflow_result(self, exec_id):
                 raise exceptions.ReferenceNotFoundError(mistral_exc.message)
             raise mistral_exc
 
-        result = jsonify.try_loads(execution.output) if execution.state in DONE_STATES else {}
+        result = jsonify.try_loads(
```

Comment:
Minor style thing - since it splits over multiple lines I would prefer to just use a normal and not short hand if.

In any case, not a blocker.